odxtools/parameters/parameter.py

```diff
@@ -40,6 +40,7 @@ class Parameter(NamedElement):
     define any non-positionable parameter types.
 
     """
+    oid: Optional[str]
```

Comment:
note that, for whatever reason, most parameters extending `POSITIONABLE-PARAM` do not exhibit an ID but an OID. AFAICS, the only parameters exhibiting IDs are `LENGTH-KEY` and `TABLE-KEY`. (why oh why?)